tests/data_access.rs

```diff
@@ -28,7 +28,7 @@ struct Loaded {
 }
 
 const TRACKABLE_MAX: u64 = 3600 * 1000 * 1000;
-const SIGFIG: u32 = 3;
+const SIGFIG: u8 = 3;
```

Comment:
Beyond the argument about what is a reasonable range for `SIGFIG`, is there a *technical* reason why we can't support more than 256 significant figures?